src/util/file/file_test.go

```diff
@@ -61,12 +61,12 @@ func TestBuildDataDirDotOk(t *testing.T) {
 	cleanDir := filepath.Clean(dir)
 	require.True(t, strings.HasSuffix(builtDir, cleanDir))
 
-	home := filepath.Clean(UserHome())
-	if home == "" {
+	gopath := os.Getenv("GOPATH")
+	if gopath == "" {
```

Comment:
When `GOPATH` is not set, Go will use `$HOME/go` as `GOPATH`, so the test will fail here.